packages/puppeteer-core/src/cdp/Accessibility.ts

```diff
@@ -259,9 +259,7 @@ export class Accessibility {
 
     const interestingNodes = new Set<AXNode>();
     this.collectInterestingNodes(interestingNodes, defaultRoot, false);
-    if (!interestingNodes.has(needle)) {
-      return null;
-    }
+
```

Comment:
Let's discuss this change, I believe the new behavior is better as the user may know the start of something and need only it's sub-tree.